src/modules/hip/kernel/gaussian_filter.hpp

```diff
@@ -0,0 +1,2065 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+// -------------------- Set 0 - gaussian_filter device helpers --------------------
+
+__device__ void gaussian_filter_3x3_row_hip_compute(uchar *srcPtr, d_float8 *dst_f8, d_float9 *filter_f9, int rowIndex)
+{
+    float src_f1;
+    uint3 src_ui3;
+    src_ui3 = *(uint3 *)srcPtr;
+    src_f1 = rpp_hip_unpack0(src_ui3.x);
+    dst_f8->f1[0] = fmaf(src_f1, filter_f9->f1[rowIndex], dst_f8->f1[0]);
+    src_f1 = rpp_hip_unpack1(src_ui3.x);
+    dst_f8->f1[0] = fmaf(src_f1, filter_f9->f1[rowIndex + 1], dst_f8->f1[0]);
```

Comment:
Please remove the add here

Reply:
Done this for 3x3 kernels. Need to replicate for other kernel sizes
https://github.com/sampath1117/rpp/blob/sr/opt_gauss_filter_hip/src/modules/hip/kernel/gaussian_filter.hpp#L6
https://github.com/sampath1117/rpp/blob/sr/opt_gauss_filter_hip/src/modules/hip/kernel/gaussian_filter.hpp#L302
https://github.com/sampath1117/rpp/blob/sr/opt_gauss_filter_hip/src/modules/hip/kernel/gaussian_filter.hpp#L339

Reply:
Replicated for remaining kernel sizes